### PR TITLE
Allow mute to be created even where there's channels that deny permission.

### PR DESCRIPTION
**Pull request by KJP12**
Allow mute to be created even where there's channels that deny permission.
Replies a list of channels that denied permission for manual editing.

### CODE REVIEW

File: src/main/java/co/protector/bot/commands/admin/MuteCommand.java [hunk 1]
```diff
@@ -9,6 +9,8 @@
 import net.dv8tion.jda.core.utils.PermissionUtil;
 import org.bson.Document;
 
+import java.util.ArrayList;
```
Comment (Repulser) [inline src/main/java/co/protector/bot/commands/admin/MuteCommand.java]: Unused import

File: src/main/java/co/protector/bot/commands/admin/MuteCommand.java [hunk 2]
```diff
@@ -116,8 +118,9 @@ private void setMutedRole(String id, String guildid) {
 
     private Role createRole(Guild guild) {
         Role role = guild.getController().createRole().setName("Muted").complete();
-        guild.getTextChannels().forEach(textChannel ->
-                textChannel.createPermissionOverride(role).setDeny(Permission.MESSAGE_WRITE).queue());
+        guild.getTextChannels().forEach(textChannel ->{
+                if(PermissionUtil.checkPermission(textChannel, guild.getSelfMember(), Permission.MANAGE_PERMISSIONS))
+                    textChannel.createPermissionOverride(role).setDeny(Permission.MESSAGE_WRITE).queue();});
```
Comment (Repulser) [inline src/main/java/co/protector/bot/commands/admin/MuteCommand.java]: Please format this code (Ctrl + alt + L on intellij) and use curly brackets in a long if statement

Comment (KJP12) [reply src/main/java/co/protector/bot/commands/admin/MuteCommand.java]: Fixed along with the unused import.

**Comment by GreggKr** [comment]
lgtm

**Pull request merged**